common/types/wire/malfeasance.go

```diff
@@ -0,0 +1,10 @@
+package wire
+
+//go:generate scalegen
+
+type InvalidPostIndexProofV1 struct {
```

Comment:
We should consider moving this type to the `malfeasance` package (along other types for malfeasance proofs)